discord/guild.py

```diff
@@ -88,6 +88,7 @@
 from .sticker import GuildSticker
 from .threads import Thread, ThreadMember
 from .user import User
+from .utils import _D, _FETCHABLE
```

Comment:
Idk if you did this because of another comment, but TypeVars should generally not be imported.